app/src/main/res/values/strings.xml

```diff
@@ -236,7 +236,7 @@
         <item><![CDATA[<a href=https://github.com/FTno>FTno</a>]]></item>
         <item><![CDATA[<a href=https://github.com/pggdt>ft42</a>]]></item>
         <item><![CDATA[<a href=https://github.com/NetoSilva>Neto Silva</a>|<a href=https://github.com/naoliv>Nelson&#160;A.&#160;de&#160;Oliveira</a>|<a href=https://github.com/efraletti>Enrico S. B. Fraletti</a>]]></item>
-        <item><![CDATA[<a href=optionalUrlHere>TODO</a>]]></item>
+        <item><![CDATA[<a href=https://github.com/Joppla>Joppla</a>]]></item>
```

Comment:
@Joppla The attribution here is whatever you'd like. Its displayed in the About dialog and the readme.

Reply:
It is oke